Sources/LeafKit/LeafParser/LeafParser.swift

```diff
@@ -200,7 +200,13 @@ internal struct LeafParser {
                     let params = try readParameters()
                     // parameter tags not permitted to have bodies
                     if params.count > 1  { group.append(.expression(params)) }
-                    else { group.append(params.first!) }
+                    ///// else { group.append(params.first!) }
+                    else {
+                      guard let firstParam = params.first else {
```

Comment:
Can you fix this in the `dump()` function on line 187 as well? It's fine to make it throwing in the process.

Reply:
done